meilisearch-auth/src/store.rs

```diff
@@ -4,12 +4,13 @@ use std::collections::HashSet;
 use std::convert::TryFrom;
 use std::convert::TryInto;
 use std::fs::create_dir_all;
-use std::ops::Deref;
+// use std::ops::Deref;
```

Comment:
If you don't use it, can you delete it please?

Reply:
Done